config/application.rb

```diff
@@ -30,5 +30,7 @@ class Application < Rails::Application
     # in config/environments, which are processed later.
 
     config.time_zone = 'Eastern Time (US & Canada)'
+
+    config.action_mailer.preview_path = Rails.root.join('app/mailers/previews')
```

Comment:
I like this location, we should apply this to other apps as well

Reply:
Turns out Zeitwerk has a complaint, though...